api/objects.proto

```diff
@@ -229,6 +229,12 @@ message Cluster {
 	// RemovedNodes is the list of nodes that have been removed from the
 	// swarm. Their certificates should effectively be blacklisted.
 	repeated RemovedNode removed_nodes = 7;
+
+	// ManagerUnlockKey defines the key that locks a manager at rest.  This is the
+	// key encrypting key (KEK) that will encrypt the manager TLS keys at rest and
+	// the raft encryption keys at rest.  If not provided, then managers will be
+	// unlocked (will not require a key to start up from a shut down state).
+	bytes manager_unlock_key = 9;
```

Comment:
We implemented join token rotation by adding flags to the `UpdateCluster` request: https://github.com/docker/swarmkit/blob/master/api/control.proto#L334

This follows the model of each object having `Get`/`Update`/`List` endpoints, and not adding miscellaneous imperative calls.

It probably makes sense to do this rotation in a similar way.